### PR TITLE
Stand First added to hover state

### PR DESCRIPTION
**Pull request by clloyd**
Improved design of the Stand First (based on https://github.com/guardian/about-to-be-published/pull/9)

screen shot 2015-08-05 at 11 20 24

@chrisfinch

### CODE REVIEW

**Comment by chrisfinch** [comment]
:tada: :+1:

File: public/components/ReactApp.jsx [hunk 1]
```diff
@@ -41,20 +41,22 @@ var DeltaItem = React.createClass({
 
                 <div className="image__container">
                     <img className="image" src={item.imageUrl} />
+                </div>
 
-                    <div className="content-item__links">
+                <div className="content-item__hoverstate">
+                  <div className="content-item__standfirst" dangerouslySetInnerHTML={{__html: item.standfirst}} />
```
Comment (chrisfinch) [inline public/components/ReactApp.jsx]: Is it necessary to do this? I take it the standfirst contains tags... is there a raw version in the response?

Comment (clloyd) [inline public/components/ReactApp.jsx]: It's not necessary if we just want the content, no functioning links etc. Can easily parse out the text from the response, would that be preferable?

**Comment by clloyd** [comment]
@chrisfinch I've removed the HTML tags

**Comment by chrisfinch** [comment]
:+1:

**Pull request merged**